service/awsconfig/v7/resource/cloudformation/delete.go

```diff
@@ -41,7 +41,7 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 	if err != nil {
 		return microerror.Maskf(err, "deleting AWS Host Post-Guest CloudFormation Stack")
 	}
-	r.logger.LogCtx(ctx, "debug", "deleting AWS Host Post-Guest CloudFormation stack: deleted")
+	r.logger.LogCtx(ctx, "debug", "deleted the guest cluster main stack")
```

Comment:
This has to be done in order to make the integration tests happy. I would like to have integration tests within the versioned folders here in the future. This would imply we have to adapt the circle config with every time we move forward but the tooling could deal with this. Thoughts? @fgimenez @kopiczko

Reply:
Having the integration tests versioned makes sense to me. The current e2e tests get the active and WIP version. Maybe the same logic can be reused for this?

Reply:
:+1: to versioned e2e tests, makes total sense. We should even aim to have e2e tests that only makes sense for a specific version, covering the new features. Not sure how the tooling could help here though.